apps/web/public/static/locales/ar/common.json

```diff
@@ -615,7 +615,7 @@
   "disable_notes": "إخفاء الملاحظات في التقويم",
   "disable_notes_description": "لأسباب تتعلق بالخصوصية، سيتم إخفاء المدخلات والملاحظات الإضافية في إدخال التقويم. سيتم إرسالها إلى بريدك الإلكتروني.",
   "opt_in_booking": "حجز يتطلب موافقة",
```

Comment:
You left the opt in booking string in, and only changed the name of the description string